variablesview.py

```diff
@@ -279,6 +283,7 @@ class VariablesView(QTreeView):
 
     def __init__(self, parent=None):
         QTreeView.__init__(self, parent)
+        self.parent = parent
```

Comment:
```suggestion
```

This isn't needed -- it's actually always available for any Qt widget via the `parent()` function.